src/targets/services/cliskTimeout.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * This service stops the given job with a 'context deadline exceeded' error. It may be called when
+ * the execution of the clisk konnector is too long or when the flagship app has crashed.
+ * The flagship app will also timeout the job if it restarts but this way, the user will not see
+ * the job as running even outside the flagship app
+ *
+ * This service is supposed to be called directly or via a @in or @at trigger
+ *
+ * Example to call this service :
+ *
+ *
+ *  await client.collection('io.cozy.jobs').create(
+ *     'service',
+ *     {
+ *       name: 'cliskTimeout',
+ *       slug: 'home',
+ *       fields: {
+ *        cliskJobId: '<the job id>'
+ *       }
+ *     }
+ *   )
+ */
```

Comment:
We don't have any mechanism to be sure that `cliskJobId` is passed on calling the creation?

Reply:
Only the flagship application checks if the job id is defined. I have no idea of any way to check parameters on client.create

Reply:
I didn't mean on the client side, but more on the server side. To not write thing like:
```
if (!cliskJobId) {
    throw new Error('No cliskJobId field defined. Could find any job to update')
  }
```

but ok, I think I have my answer;)